apps/web/src/utils/batcher.ts

```diff
@@ -13,5 +13,8 @@ export const usdPriceBatcher = create<CurrencyUsdResult, CurrencyParams, number>
     return items[key] ?? 0
   },
   // this will batch all calls to users.fetch that are made within 60 milliseconds.
-  scheduler: windowScheduler(60),
+  scheduler: windowedFiniteBatchScheduler({
```

Comment:
duplicated schedule config, can we define a global config to reuse

Reply:
will modify this value after all features done for better performance